README.md

```diff
@@ -47,24 +56,49 @@ val mapper = ObjectMapper()
 
 data class Foo(val value: Option<String>)
 data class Bar(val value: Nel<String>)
+data class Baz(val value: Either<Int, String>)
```

Comment:
These are great additions and build on a previous example. The issue here is the last model example is not great. Can we rethink these docs to include a more meaningful model that does not use `Foo` etc.? After all, this would land in public docs. Thanks!

Reply:
I agree, i think it's probably a better idea if we use some real data structure as an example. Something like this for instance.
```kotlin
data class ArrowUser(
  val id: String,
  val emails: NonEmptyList<EmailAddress>,
  val website: Option<String>
)
```

Illustrating `Either`, `Ior` and `Validated` would be a little strange. I guess we can simply mention that they are supported.

I'll try to come up with something.

Reply:
I've updated the example in readme to reflect that.
```kotlin
val mapper = ObjectMapper()
  .registerKotlinModule()
  .registerArrowModule()
  .setSerializationInclusion(JsonInclude.Include.NON_ABSENT) // will not serialize None as nulls

data class Organization(val name: String, val address: Option<String>, val websiteUrl: Option<URI>)
data class ArrowUser(val name: String, val emails: NonEmptyList<String>, val organization: Option<Organization>)

val arrowKt = Organization("arrow-kt", none(), URI("https://arrow-kt.io").some())
val arrowUser = ArrowUser(
  "John Doe",
  nonEmptyListOf("john@email.com", "john.doe@email.com.au"),
  arrowKt.some()
)

val prettyPrinter = mapper.writerWithDefaultPrettyPrinter()
prettyPrinter.writeValueAsString(user)
// {
//   "name" : "John Doe",
//   "emails" : [ "john@email.com", "john.doe@email.com.au" ],
//   "organization" : {
//     "name" : "arrow-kt",
//     "websiteUrl" : "https://arrow-kt.io"
//   }
// }

val validOrganization: Validated<Nel<String>, Organization> = arrowKt.valid()
prettyPrinter.writeValueAsString(validOrganization)
// {
//   "valid" : {
//     "name" : "arrow-kt",
//     "websiteUrl" : "https://arrow-kt.io"
//   }
// }
```

Reply:
Also added the documentation on how to register it in Spring Boot